internal/cli/alpha/manifestgen/templates.go

```diff
@@ -0,0 +1,340 @@
+          revision: 0.1.0
+`
+
+	typeManifestTemplate = `ocfVersion: 0.0.1
+revision: 0.1.0
+kind: Type
+metadata:
+  prefix: "cap.type.{{ .Prefix }}"
+  name: {{ .Name }}-input
+  displayName: "Input for {{ .Prefix }}.{{ .Name }}"
+  description: Input for the "{{ .Prefix }}.{{ .Name }} Action"
+  documentationURL: https://example.com
+  supportURL: https://example.com
+  maintainers:
+    - email: dev@example.com
+      name: Example Dev
+      url: https://example.com
+spec:
+  jsonSchema:
+    value: |-
+      {
+        "$schema": "http://json-schema.org/draft-07/schema",
+        "type": "object",
+        "required": [],
+        "properties": {
+          {{ $length := len .Variables -}}
+          {{ range $index, $variable := .Variables -}}
+          "{{ $variable.Name }}": {
+            "$id": "#/properties/{{ $variable.Name }}",
+            "type": "{{ $variable.Type }}",
+            "description": "{{ $variable.Description }}"{{if $variable.Default }},
+            "default": "{{ $variable.Default }}"{{ end }}
+          }{{ if ne $index (add $length -1) }},{{ end }}
+          {{ end }}
+        }
+      }
+`
+
+	outputTypeManifestTemplate = `ocfVersion: 0.0.1
+revision: 0.1.0
+kind: Type
+metadata:
+  prefix: "cap.type.{{ .Prefix }}"
+  name: config
+  displayName: "{{.Prefix }} config"
+  description: "Type representing a {{ .Prefix }} config"
+  documentationURL: https://example.com
+  supportURL: https://example.com
+  maintainers:
+    - email: dev@example.com
+      name: Example Dev
+      url: https://example.com
+spec:
+  jsonSchema:
+    # Put the properties of your Interface output Type in form of a JSON Schema here:
+    value: |-
+      {
+        "$schema": "http://json-schema.org/draft-07/schema",
+        "type": "object",
+        "required": [],
+        "properties": {
+          "example": {
+            "$id": "#/properties/example",
+            "type": "string",
+            "description": "Example field"
+          }
+        }
+      }
+`
+
+	terraformImplementationManifestTemplate = `ocfVersion: 0.0.1
+revision: {{ .Revision }}
+kind: Implementation
+metadata:
+  prefix: "cap.implementation.{{ .Prefix }}"
+  name: {{ .Name }}
+  displayName: "{{ .Name }} Action"
+  description: "{{ .Name }} Action"
+  documentationURL: https://example.com
+  supportURL: https://example.com
+  maintainers:
+    - email: dev@example.com
+      name: Example Dev
+      url: https://example.com
+  license:
+    name: "Apache 2.0"
+
+spec:
+  appVersion: "1.0.x" # Set the supported application version here
+  additionalInput:
+    parameters:
+      typeRef:
+        path: "cap.type.{{ .Prefix }}.{{ .Name }}-input"
+        revision: 0.1.0
+
+  outputTypeInstanceRelations:
+    config:
+      uses:
+        - terraform-release
+
+  implements:
+    - path: {{if .InterfacePath}}cap.interface.{{ .InterfacePath }}{{else}}"cap.interface..." # Put here the path of the implemented Interface{{end}}
```

Comment:
Using full paths everywhere will be more consistent.

```suggestion
    - path: {{if .InterfacePath}}{{ .InterfacePath }}{{else}}"cap.interface..." # Put here the path of the implemented Interface{{end}}
```

I ran:

```
go run cmd/cli/main.go alpha manifest-gen implementation terraform cap.implementation.aws.iam.user.create ../terraform-aws-iam/modules/iam-user -i cap.interface.cloud.user.create -s "https://storage.googleapis.com/capactio-terraform-modules/terraform.aws.iam.user/0.1.0/module.tgz" -p aws
```